src/main/java/com/onaple/itemizer/data/serializers/IngredientMapAdapter.java

```diff
@@ -24,6 +25,10 @@
             String s = entry.getKey().toString();
             ConfigurationNode node = entry.getValue();
             ItemStack item = new ItemBeanRefOrItemIdAdapter().deserialize(type, node.getNode("item"));
+            if (item == null) {
+                Itemizer.getLogger().warn("Unknown item " + node.getNode("item").toString());
+                continue;
```

Comment:
(pro tips I ve learned at work) the best way to use Slf4j is to write the message with {} in the first arg, and objects you want to log in others args. 😉